apps/web/modules/ui/components/id-badge/index.tsx

```
@@ -0,0 +1,80 @@
+            onMouseLeave={() => setIsHovered(false)}>
+            <span>
+              {prefix ? `${prefix} ` : ""}
+              {id}
+            </span>
+
+            {shouldShowIcon && (
+              <button
+                onClick={handleCopy}
+                className="flex h-4 w-4 items-center justify-center rounded transition-colors"
+                title={t("common.copy")}
+                aria-label={`Copy ${prefix ? `${prefix} ` : ""}${id}`}>
```

Biome (1.9.4)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)